telegraf-kubernetes.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+import sys
+import argparse
+from kubernetes import Client, Parser
+
+def __handle_cli(args):
+    parser = argparse.ArgumentParser(
+        description='Plugin for Telegraf for gathering statistics from Kubernetes')
+
+    parser.add_argument(
+        '--stats-url',
+        help='Kubernetes stats URL',
+        default="http://127.0.0.1:10255/stats/summary",
```

Comment:
WDYT to use only IP address and port here? 
As I know `/stats/summary` it's API contract